src/test/resources/org/openmrs/uitestframework/test.properties

```diff
@@ -2,7 +2,7 @@ webapp.url=https://qa-refapp.openmrs.org/openmrs
 login.username=admin
 login.password=Admin123
 login.location=Pharmacy
-webdriver=firefox
+webdriver=chrome
 login.auto=false
-headless=true
-webdriver.gecko.driver=firefoxdriver/linux/geckodriver
\ No newline at end of file
+headless=false
+webdriver.gecko.driver=chromedriver/mac/geckodriver
```

Comment:
any reason for changing values in `test.properties` file?

Reply:
@kdaud am sorry I forgot and added to the commit when it was just mean't to help me run the test.